migrations/versions/590_remove_brief_status_column.py

```diff
@@ -0,0 +1,22 @@
+"""Remove brief status column
+
+Revision ID: 590
+Revises: 580
+Create Date: 2016-03-03 14:56:59.218753
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '590'
+down_revision = '580'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.drop_column('briefs', 'status')
+
+
+def downgrade():
+    op.add_column('briefs', sa.Column('status', sa.VARCHAR(), autoincrement=False, nullable=False))
```

Comment:
I don't think this downgrade will work if there are any briefs because there's no default value set.  I think the way to do it would be to add the column as `nullable=True`, run a query to populate the status for existing briefs (depending on published time, etc.), and then set the column to `nullable=False`.